addons/osio-addon/src/test/resources/jenkinsfile-preparer-test/pipeline/inject/Jenkinsfile

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/groovy
+@Library('github.com/fabric8io/fabric8-pipeline-library@master')
+def canaryVersion = "1.0.${env.BUILD_NUMBER}"
+
+// INJECT FRAGMENT
```

Comment:
The fragment needs a name and the name is the same that `Jenkinsfile.<name>.fragment` uses

Reply:
Although I think we could call `SNIPPET` instead of fragment

Reply:
on my fragment inject PR you didn't want to call it `config-map` https://github.com/fabric8io/fabric8-jenkinsfile-library/pull/28

Reply:
Yes, I wanted it to be called `pre-deploy` or something like that (like a hook name)